packages/spec/src/locales.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { describe, it, expect } from "vitest";
-import { normalizeLocale } from "./locales";
+import { getLocaleCodeDelimiter, normalizeLocale, resolveLocaleCode, resolveOverridenLocale } from "./locales";
```

Comment:
@khalatevarun the `resolveOverridenLocale` function was renamed in #576 to `resolveOverriddenLocale`. Once this is resolved we can merge it 🙌